pkg/cloudprovider/backup_cache_test.go

```diff
@@ -45,32 +45,32 @@ func TestNewBackupCache(t *testing.T) {
 		test.NewTestBackup().WithName("backup1").Backup,
 		test.NewTestBackup().WithName("backup2").Backup,
 	}
-	delegate.On("GetAllBackups", "bucket1").Return(bucket1, nil).Once()
+	delegate.On("GetAllBackups", "bucket1", "path1").Return(bucket1, nil).Once()
 
 	// should be updated via refresh
 	updatedBucket1 := []*v1.Backup{
 		test.NewTestBackup().WithName("backup2").Backup,
 	}
-	delegate.On("GetAllBackups", "bucket1").Return(updatedBucket1, nil)
+	delegate.On("GetAllBackups", "bucket1", "path1").Return(updatedBucket1, nil)
 
 	// nothing in cache, live lookup
 	bucket2 := []*v1.Backup{
 		test.NewTestBackup().WithName("backup5").Backup,
 		test.NewTestBackup().WithName("backup6").Backup,
 	}
-	delegate.On("GetAllBackups", "bucket2").Return(bucket2, nil).Once()
+	delegate.On("GetAllBackups", "bucket2", "path2").Return(bucket2, nil).Once()
```

Comment:
All of the paths should be identical for a given test or set of tests. Right now Ark only supports a single configuration, so it doesn't make sense to vary the path.